packages/flutter_app/lib/util/providers/app_lifecycle_provider.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/widgets.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'app_lifecycle_provider.g.dart';
+
+@riverpod
```

Comment:
I removed `keepAlive:true` since it only needs to survive where I want to monitor.